api/config/config_local.json

```diff
@@ -123,7 +123,6 @@
   },
   "reporting": {
     "dataset": "reporting_local",
-    "maxRowsPerInsert": 200,
-    "uploadMethod": "STREAMING"
```

Comment:
When was `uploadMethod` last used in the code?  Please ensure that there is at least one release between that merge and this one.

Reply:
🔪  https://github.com/all-of-us/workbench/pull/4355/files#diff-fd748e8599d99102a8da44afd79c7e9483084d1857c5bf983e4252a9b5243087L66

Reply:
Good point. I'll leave this in place for rollback safety.